### ISSUE TITLE
zero-shot setting

### ISSUE THREAD

**Issue by liuc-v** [yeliudev/ConsNet#17]
Hi,in the zero sample setting, do you set all pictures containing unseen hoi to invisible, regardless of whether the picture also contains a seen hoi sample. Where's the corresponding code. Thanks

**Comment by yeliudev** [comment]
You are correct. Please refer to [this line](https://github.com/yeliudev/ConsNet/blob/main/consnet/datasets/hico_det.py#L28:L31).
